packages/web/scripts/grant-free-subscription.ts

```diff
@@ -39,8 +39,6 @@ const handler = async (userId: number, months: number | undefined) => {
       expiresAt: expiresAt,
       stripeSubscription: stripeSubscription as unknown as Prisma.InputJsonValue,
       stripeSubscriptionId: stripeSubscription.id,
-      lastFourCardNumbers: 'NONE',
-      cardBrand: 'visa',
```

Comment:
We should still apply this change to the user so we can present values (FE may choke if they have a subscription but these are null, not sure)

Reply:
Great call 👍🏼